### DISCUSSION TITLE
docs/service-mesh/

### DISCUSSION THREAD

**Discussion by giscus** [ricsanfre/pi-cluster#419] [Announcements]
# docs/service-mesh/

How to deploy service-mesh architecture based on Linkerd. Adding observability, traffic management and security to our Kubernetes cluster.

https://picluster.ricsanfre.com/docs/service-mesh/

**Comment by benbourner** [comment]
Two corrections needed in `linkerd-prometheus.yml` above to make it valid:
1) `source_labels` -> `sourceLabels`
2) `relabellings` -> `- relabellings` (podMetricsEndpoints should be an array)
